### PR TITLE
Fix issue 92

### PR DESCRIPTION
**Pull request by ragabala**
This commit takes care of the following                                                                                                                                                                                                                                
 * It adds a new errata to an existing repo with erratum present
 * Makes updates to this new errata and checks whether it gets updated as
 required
 See #92

### CODE REVIEW

**Comment by kersommoura** [comment]
@ragabala, please rebase this PR.

**Comment by ragabala** [comment]
Hi @kersommoura,
Have rebased the issue. Can you check this once so this can be merged.

Thanks!!

File: pulp_2_tests/tests/rpm/api_v2/test_updateinfo.py [hunk 1]
```diff
@@ -200,28 +202,29 @@ def do_test(self, date):
...
     def test_03_newer_date(self):
-        """Create errata using a newer valid date. See :meth:`do_test`."""
+        """Create errata using a newer valid date. See:meth:`do_test`."""
         updates = self.do_test(ERRATA_UPDATE_INFO['new_updated_date'])
         self.assertEqual(len(updates), 1, updates)
         self.assertEqual(updates[0], ERRATA_UPDATE_INFO['new_updated_date'])
 
     def test_04_invalid_date(self):
-        """Attempt to upload an errata with an invalid date."""
+        """Attemp to upload an errata with an invalid date."""
```
Comment (kersommoura) [inline pulp_2_tests/tests/rpm/api_v2/test_updateinfo.py]: Those docstrings changes are already present on the code basis. [docs](https://pulp-2-tests.readthedocs.io/en/latest/tests/pulp_2_tests.tests.rpm.api_v2.test_updateinfo.html#pulp_2_tests.tests.rpm.api_v2.test_updateinfo.DateUpdateErrataTestCase)

**Comment by ragabala** [comment]
Hi @kersommoura ,

Thanks for those reviews. Have made the changes.

Let me know if this looks fine.

**Pull request merged**